modules/ravestate_nlp/question_words.py

```diff
@@ -0,0 +1,27 @@
+from spacy.tokens import Token
+
+
+class QuestionWord:
+    FORM = 'FORM'
+    CHOICE = 'CHOICE'
+    OBJECT = 'OBJECT'
+    REASON = 'REASON'
+    TIME = 'TIME'
+    PERSON = 'PERSON'
+    PLACE = 'PLACE'
+
+    QUESTION_POS = 'QUESTION'
+    QUESTION_WORDS = {
+        'where': PLACE,
+        'who': PERSON,
+        'when': TIME,
+        'why': REASON,
+        'what': OBJECT,
+        'which': CHOICE,
+        'HOW': FORM
```

Comment:
All the other question words are lower case... this one seems to more important ;) :-*

Reply:
lol... story of my life 🤦‍♀️

Reply:
Also, shouldn't it be MANNER?

Reply:
Can use both 🐳  I used this professional website for information: http://www.grammar.cl/Notes/Question_Words.htm